README.md

```diff
@@ -64,11 +64,11 @@ DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:<release>/lib/native/osx`
 
 ### Windows x86
 
-`PATH=$PATH;<release>\lib\native\windows;<release>\lib\native\windows32;<release>\lib\native\32`
+`PATH=$PATH:<release>\lib\native\windows;<release>\lib\native\windows32;<release>\lib\native\32`
```

Comment:
note that this was correct before. the path separator on windows outside of a bash-like environment is in fact a semicolon.

Reply:
Oh my bad. Sorry